tests/suites/cloud/conf.js

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
This file is no longer needed. We need to update Leviathan in all working testbots to get this change in.

Reply:
all testbots are now at latest, including the conf.js changes